endoreg_db/models/upload_job.py

```
@@ -0,0 +1,100 @@
+import uuid
+from django.db import models
+from django.conf import settings
+
+
+class UploadJob(models.Model):
+    """
+    Tracks file upload jobs and their processing status.
+    Supports both PDF and video file uploads with asynchronous processing.
+    """
+    
+    class Status(models.TextChoices):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)